codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/protocols/restxml/RestXMLProtocolGenerator.kt

```diff
@@ -14,19 +14,18 @@ class RestXMLProtocolGenerator : AWSHTTPBindingProtocolGenerator(RestXMLCustomiz
     override val defaultContentType: String = "application/xml"
     override val protocol: ShapeId = RestXmlTrait.ID
     override val testsToIgnore: Set<String> = setOf(
-        "S3DefaultAddressing",
-        "S3VirtualHostAddressing",
-        "S3VirtualHostDualstackAddressing",
-        "S3VirtualHostAccelerateAddressing",
-        "S3VirtualHostDualstackAccelerateAddressing",
-        "S3OperationAddressingPreferred",
-        "S3EscapeObjectKeyInUriLabel",
-        "S3EscapePathObjectKeyInUriLabel",
-        "SDKAppliedContentEncoding_restXml",
-        "SDKAppendedGzipAfterProvidedEncoding_restXml",
-        "S3OperationNoErrorWrappingResponse",
-        "S3PreservesEmbeddedDotSegmentInUriLabel",
-        "S3PreservesLeadingDotSegmentInUriLabel",
+        "S3DefaultAddressing", // can leave disabled, pre-endpoints 2.0
+        "S3VirtualHostAddressing", // can leave disabled, pre-endpoints 2.0
+        "S3VirtualHostDualstackAddressing", // can leave disabled, pre-endpoints 2.0
+        "S3VirtualHostAccelerateAddressing", // can leave disabled, pre-endpoints 2.0
+        "S3VirtualHostDualstackAccelerateAddressing", // can leave disabled, pre-endpoints 2.0
+        "S3OperationAddressingPreferred", // can leave disabled, pre-endpoints 2.0
+        "S3EscapeObjectKeyInUriLabel", // moved to s3-tests.smithy
+        "S3EscapePathObjectKeyInUriLabel", // moved to s3-tests.smithy
+        "SDKAppliedContentEncoding_restXml", // not implemented yet (request compression)
+        "SDKAppendedGzipAfterProvidedEncoding_restXml", // not implemented yet (request compression)
+        "S3PreservesEmbeddedDotSegmentInUriLabel", // moved to s3-tests.smithy
+        "S3PreservesLeadingDotSegmentInUriLabel", // moved to s3-tests.smithy
```

Comment:
Tests that have been moved to s3-tests.smithy would still remain ignored here for when they are pulled from smithy. They have been copied and renamed.

Reply:
If tests that have been copied over to `s3-tests.smithy` are modified or added to, how will we know (other than manually checking those test models)?

Reply:
they'll continue to be marked as disabled here with a comment noting they've been moved. Another place to check is in AdditionalServiceTests/<service> folder for the smithy file